samples/testing-frameworks/appium/client-side/python/tests/bitbar_ios.py

```diff
@@ -16,13 +16,13 @@ def setUp(self):
         if self.bitbar_device == "":
             self._find_device("iOS")
 
-        self.desired_capabilities_cloud["bitbar_project"] = bitbar_project_name
-        self.desired_capabilities_cloud["bitbar_device"] = self.bitbar_device
-        self.desired_capabilities_cloud["bundleId"] = bitbar_bundle_id
         self.desired_capabilities_cloud["platformName"] = "iOS"
-        self.desired_capabilities_cloud["deviceName"] = "iOS device"
-        self.desired_capabilities_cloud["bitbar_target"] = "ios"
-        self.desired_capabilities_cloud["automationName"] = automation_name
+        self.desired_capabilities_cloud["bitbar:options"]["project"] = bitbar_project_name
+        self.desired_capabilities_cloud["bitbar:options"]["device"] = self.bitbar_device
+        self.desired_capabilities_cloud["bitbar:options"]["target"] = "ios"
+        self.desired_capabilities_cloud["appium:bundleId"] = bitbar_bundle_id
+        self.desired_capabilities_cloud["appium:automationName"] = automation_name
```

Comment:
for Android you declared driver directly. Is there any reason for allowing changing of that value with environmental variable?

Reply:
I don't know tbh. Just left it as it was previously. However currently there are actually no automations to choose from on Android with appium 2.0 so I guess it's ok.